game/server/eventstream/models.py

```diff
@@ -56,6 +59,10 @@ def from_events(
                 FrozenCombatJs.for_combat(c, game.theater)
                 for c in events.updated_combats
             ]
+            if events.updated_supply_routes:
+                updated_supply_routes = SupplyRouteJs.all_in_game(game)
+            for route in updated_supply_routes:
+                route.points = []
```

Comment:
This seems to be two distinct events masquerading as one, only distinguished by whether or not `points` is empty? This makes both this spot and the reducer awkward. I think you have both an update event (which is what this is attempting to do, which only updates non-coordinate state), and an add event for game load? Why not just make two events?

Reply:
At this point all supply routes get updated whenever a change in one is detected back-end wise. As you point out, it's only the state that changes and not the actual route, which is why I delete the points.
I plan on reworking the back-end to push single routes instead of them all. This may not be the only component to have this problem but I'll have to review it once most merges have happened.

Reply:
But why not just make two models?